sda/cmd/ingest/ingest_test.go

```diff
@@ -102,9 +130,14 @@ func (suite *TestSuite) TestTryDecrypt() {
 	buf, err := io.ReadAll(file)
 	assert.NoError(suite.T(), err)
 
-	key, err := config.GetC4GHKey()
-	assert.NoError(suite.T(), err)
-	header, err := tryDecrypt(key, buf)
+	keyList, err := config.GetC4GHprivateKeys()
 	assert.NoError(suite.T(), err)
-	assert.NotNil(suite.T(), header)
+	for _, key := range keyList {
+		header, err := tryDecrypt(key, buf)
+		if header != nil && err == nil {
+			break
+		}
+		assert.NoError(suite.T(), err)
+		assert.NotNil(suite.T(), header)
+	}
```

Comment:
```suggestion
		header, err := tryDecrypt(key[0], buf)
		assert.NoError(suite.T(), err)
		assert.NotNil(suite.T(), header)
```

No iteration is needed since the test only has one key